book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -217,7 +218,7 @@ Only if this section is defined a sensitivity analysis is done. This section wil
 * **end.date** : [required?] end date of the sensitivity analysis (in YYYY/MM/DD format)
 * **_NOTE:_** start.date and end.date are distinct from values set in the run tag because this analysis can be done over a subset of the run.
 * ** variable** : [optional] name of one (or more) variables the analysis should be run for. If not specified, sensitivity.analysis variable is used, otherwise default is GPP.
-
+* ** perpft** : [optional] if `TRUE` a sensitivity analysis on PFT-specific outputs will be run. This is only possible if your model provides PFT-specific outputs for the `variable` requested. 
```

Comment:
Do we really want the default for this to be TRUE? For some models that could cause a very large number of runs to be started. It seems to me that we'd want to default this to FALSE.

Reply:
I'm confused, this tag doesn't change the number of sensitivity runs, just enables the reading of the outputs per pft. Current SA workflow by default does start a larger number of runs if there are larger number of PFTs, no?

Reply:
Ok, that's not clear from the documentation you wrote. I'd recommend updating to make it clear that this only affects the output processing, not the execution.

I expect/hope the need for this flag to be deprecated in the not-to-distant future, since we really should be doing the output post-processing and analysis on the fly (e.g. in the Shiny apps) rather than requiring that it all be specified up front.